Package.swift

```diff
@@ -14,11 +14,13 @@ let package = Package(
     .visionOS(.v1),
   ],
   products: [
-    .executable(name: "safedi-formatter", targets: ["CLI"]),
+    .executable(name: "safedi-formatter", targets: ["CLI"])
```

Comment:
I ran swift format on this (and all the other) files. So there were some unnecessary format changes in this PR. Sorry about that!